docs/source/information_model.rst

```diff
@@ -76,6 +76,28 @@ coordinate. Reference sequences used to describe Sequence Locations should be ve
      - 1..1
      - A coordinate representing the end of a genomic location.
 
+.. _reading-frame:
+
+Reading Frame
+#############
+A common attribute of a gene fusion is whether the reading frame is preserved in the expressed gene
+product (for categorical fusions) or whether it is predicted to be preserved based on assayed
+findings (for assayed fusions). This is typical of protein-coding gene fusions.
+
+.. list-table::
+   :class: clean-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Limits
+     - Description
+   * - Reading frame preserved
+     - 0..1
+     - Boolean indicating whether the reading frame must be preserved, or is predicted to be preserved.
```

Comment:
This should be reworded; it should be a Boolean representing if the reading frame is preserved or not preserved. If false, reading frame is not preserved. The difference is not must / should.

Reply:
Gotcha. I was trying to convey that `true` means "must be, or is predicted to be preserved" but obviously better if we can just say "is"